core/ikonli-javafx/src/main/java/org/kordamp/ikonli/javafx/FontIcon.java

```diff
@@ -17,6 +17,17 @@
  */
 package org.kordamp.ikonli.javafx;
 
+import static java.util.Collections.unmodifiableList;
```

Comment:
Please revert imports to their previous location. It looks to me you're using code formatting settings that are not the same as Intellij's